src/samtools/samtools_view/config.vsh.yaml

```diff
@@ -0,0 +1,351 @@
+name: samtools_view
```

Comment:
According to the CI, there is some utf8 characters in this file. We tend to avoid them, since a fairly recent version of Java had a bug that disallowed us to parse UTF8 (if I recall correctly)

Reply:
Ah yes I was wondering why the tests were not passing, I will look for the issue, thanks.